src/popup/components/AccountCardConsensus.vue

```diff
@@ -27,7 +27,11 @@
 </template>
 
 <script lang="ts">
-import { computed, defineComponent, PropType } from '@vue/composition-api';
+import {
+  computed, defineComponent, PropType,
+} from 'vue';
```

Comment:
No need to split this line. But If you think it's a good thing I suggest to split each of items into separate lines.